cinder/autogenerated_client.go

```diff
@@ -1,14 +1,11 @@
 // Copyright 2015 Canonical Ltd.
 // Licensed under the LGPLv3, see LICENCE file for details.
 
-// This file is not intended to be updated by hand, rather it should
-// be regenerated every time this library is built. The WADLs were
-// originally sourced from:
+// This file was initially autogenerated from the upstream wadl file, but has
```

Comment:
This makes me sad, but the WADL format really isn't complete enough to auto-generate everything anyway. Pity.